packages/v/vcpkg/xmake.lua

```diff
@@ -0,0 +1,27 @@
+package("vcpkg")
+    set_homepage("https://github.com/microsoft/vcpkg")
+    set_description("Vcpkg helps you manage C and C++ libraries on Windows, Linux and MacOS.")
+    set_license("MIT")
+
+    add_urls("https://github.com/microsoft/vcpkg/archive/refs/tags/$(version).tar.gz")
+    add_versions("2024.05.24", "3034e534d4ed13e6e6edad3c331c0e9e3280f579dd4ba86151aa1e2896b85d31")
+
+    add_deps("unzip", "cmake", "ninja", "libcurl")
+    if linuxos.name() == "archlinux" or linuxos.name() == "manjaro" then
+        add_deps("pacman::zip", "pacman::curl")
```

Comment:
we should not use add_deps for 3rd packages.

Reply:
Because there is no zip command in the workflow, and there isn't one in xmake either, it was added. If it is removed, Arch Linux will not be supported.

Reply:
so you should add zip package to xmake-repo first and use `add_deps("zip")`, 

please do not add any pacman package deps.

Reply:
but,xrepo hava not a package names `zip`

Reply:
> but,xrepo hava not a package names `zip`

so you need open a pr to add it.

Reply:
这个无能为力啊.十几年前的老包了,干不动

Reply:
This is beyond my capability. It's an old package from over a decade ago and I can't handle it.

Reply:
There's also the unzip package, providing unzip.exe, wouldn't that be enough?

Reply:
@SirLynix unzip.exe,not enough,I had already tested.

Reply:
https://github.com/xmake-io/xmake-repo/pull/4231

Reply:
And we need also add curl command for libcurl package. or add a new curl package